Makefile

```diff
@@ -1,5 +1,5 @@
 WEBROOT=~/src/github.com/kubernetes/website
-K8SROOT=~/src/github.com/kubernetes/kubernetes
+K8SROOT=~/k8s/src/k8s.io/kubernetes
```

Comment:
I was able to build the API ref with the original K8SROOT setting, though, I had to change to the k8s.io/kubernetes to build the kubectl reference (for example, src/k8s.io/kubernetes).